docs/upstream.md

```diff
@@ -42,15 +42,20 @@ upstream        https://github.com/AlaskaAirlines/auro-lockup.git (push)
 
 #### Sync upstream
 
-To sync your master brach with your upstream master branch, there are two ways you can do this.
-
-One way is to fetch and merge. This will work if your `main` branch shares the same history as the upstream brach sans merged updates since fork or last sync.
+To sync your `main` brach with the upstream `main` branch, there are two ways you can do this. The preferred method is to fetch and rebase. Please do not merge down.
```

Comment:
```suggestion
To sync your `main` branch with the upstream `main` branch, there are two ways you can do this. The preferred method is to fetch and rebase. Please do not merge down.
```

Reply:
This probably is in the generator, @blackfalcon.

Reply:
This is a typo in the upstream template in the generator. I addressed the typo and added it to the DoD branch.